src/release/options.d

```diff
@@ -52,8 +55,11 @@ Options parseOpts ( string[] opts )
     auto help_info = getopt(opts,
            "log|l", format("Set the logging level, one of %s",
                            iota(LogLevel.min, LogLevel.max).map!LogLevel), &options.logging,
-           "verbose", "Set logging to verbose", &verbose);
+           "verbose", "Set logging to verbose", &verbose,
 
+           "release_subsequent|f",
```

Comment:
I chose `-f` short for `follow`, as in, follow the merges and release. I'll go with release all